src/CSET/operators/_utils.py

```diff
@@ -64,3 +64,57 @@ def get_cube_yxcoordname(cube: iris.cube.Cube) -> tuple[str, str]:
         raise ValueError("Could not identify a unique y-coordinate in cube")
 
     return (y_coords[0], x_coords[0])
+
+
+def _is_transect(cube: iris.cube.Cube) -> bool:
```

Comment:
```suggestion
def is_transect(cube: iris.cube.Cube) -> bool:
```
As `_utils` is a private module, we don't need the functions within to be marked as private too.